PowerShellEditorServices.build.ps1

```diff
@@ -135,7 +135,8 @@ function Get-NugetAsmForRuntime {
 
     $packageDirPath = Join-Path $tmpDir "$PackageName.$PackageVersion"
     if (-not (Test-Path $packageDirPath)) {
-        $tmpNupkgPath = Join-Path $tmpDir 'tmp.zip'
+        $guid = New-Guid
+        $tmpNupkgPath = Join-Path $tmpDir "$guid.zip"
```

Comment:
You should probably do the proper clean up here too.

Reply:
like after you expand-archive

Reply:
Done!